fastapi_sqla/aws_rds_iam_support.py

```diff
@@ -6,33 +6,25 @@
     boto3_installed = False
     boto3_installed_err = str(err)
 
-from pydantic import BaseSettings
 from sqlalchemy import event
+from sqlalchemy.engine import Engine
 
 
-def setup(engine):
-    config = Config()
+def setup():
```

Comment:
will [this](https://github.com/dialoguemd/notepad/pull/22/files#diff-4a04c50b1e50e5ad153b61f8fd77103944ce3d94717ccc30f19ed4086caee9a2R40) still work without the engine?

Reply:
I believe it will. let me check.

Reply:
Yes, `connectable` is an instance of `sqlalchemy.engine.Engine`, and this PR makes the code tio listen to all `"do_connect"` event from any `Engine` instance

Reply:
Won't this cause problems if someone is using multiple sqlalchemy engines, such as having an RDS engine, and maybe a sqlite or something for caching?

Is it possible that we may have prioritized aesthetics over functionality. 

Since I don't foresee Dialogue ever running into this scenario I do not mind this PR at all.